app-tasks/rf/src/rf/models/scene.py

```diff
@@ -12,8 +12,9 @@ class Scene(BaseModel):
 
     def __init__(self, organizationId, ingestSizeBytes, visibility, tags,
                  datasource, sceneMetadata, name, thumbnailStatus, boundaryStatus,
-                 status, metadataFiles, sunAzimuth=None, sunElevation=None, cloudCover=None, acquisitionDate=None,
-                 id=None, thumbnails=None, tileFootprint=None, dataFootprint=None, images=None):
+                 ingestStatus, metadataFiles, sunAzimuth=None, sunElevation=None,
```

Comment:
I believe that this breaks a few things later on in this file, I can't add a comment there, but as an example, at L51 we refer to `status` which needs to be changed to `ingestStatus`.